python/nav/alertengine/dispatchers/jabber_dispatcher.py

```diff
@@ -16,12 +16,16 @@
 #
 """Plugin module for sending jabber alerts"""
 
-import xmpp
 import logging
 import time
 from threading import Thread
 from time import sleep
 
+try:
+    import xmpp
+except ImportError:
+    pass
+
```

Comment:
What happens if something now tries to use xmpp? There's nothing about it in this commit. You could try importing it where it is used, then fail out properly then, or set a setting "XMPP_AVAILABLE" somewhere else entirely that the jabber alerts then use (and a status-page in the webui could also use it maybe)